test/mocha.opts

```diff
@@ -3,4 +3,5 @@
 --check-leaks
 --timeout 10000
 --require ./test/common
+--require ./test/promisePolyfill
```

Comment:
It was necessary to include it for the documentation tests. And yeah

> I guess it would be a great deal nicer to polyfill the native promises with RSVP for the tests

Agreed :)